### PR TITLE
[DO NOT MERGE] Initial spike for recurring lambda

### PR DESCRIPTION
**Pull request by RyanMacG**

### CODE REVIEW

**Comment by adrianclay** [comment]
Did you mean to commit the zip ?

File: govwifi-lambda/lambda.tf [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+resource "aws_cloudwatch_event_rule" "every_five_minutes" {
+  name                = "every-five-minutes"
+  description         = "Fires every five minutes"
+  schedule_expression = "rate(5 minutes)"
+}
+
+resource "aws_cloudwatch_event_target" "delete_users_every_five_minutes" {
+  rule      = "${aws_cloudwatch_event_rule.every_five_minutes.name}"
+  target_id = "check_foo"
```
Comment (adrianclay) [inline govwifi-lambda/lambda.tf]: What is this?

File: govwifi-lambda/lambda.tf [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+resource "aws_cloudwatch_event_rule" "every_five_minutes" {
+  name                = "every-five-minutes"
+  description         = "Fires every five minutes"
+  schedule_expression = "rate(5 minutes)"
+}
+
+resource "aws_cloudwatch_event_target" "delete_users_every_five_minutes" {
+  rule      = "${aws_cloudwatch_event_rule.every_five_minutes.name}"
+  target_id = "test_lambda"
+  arn       = "${aws_lambda_function.test_lambda.arn}"
```
Comment (afoldesi-gds) [inline govwifi-lambda/lambda.tf]: user_deletion.arn

File: govwifi-lambda/lambda.tf [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+resource "aws_lambda_permission" "allow_cloudwatch_to_call_delete_users" {
+  statement_id  = "AllowExecutionFromCloudWatch"
+  action        = "lambda:InvokeFunction"
+  function_name = "${aws_lambda_function.test_lambda.function_name}"
```
Comment (afoldesi-gds) [inline govwifi-lambda/lambda.tf]: user_deletion

**Pull request closed without merging**